docs/content/compatible-registries.md

```diff
@@ -19,13 +19,13 @@ out a particular registry and know that it will work for you.
 | Docker Hub | Yes | Yes |
 | Digital Ocean | Yes | Yes |
 | ECR | No |  |
-| GCR | Yes |  
-| GitHub Packages | No |  | 
+| GCR | Yes |
+| GitHub Packages | Yes | Yes |
 | Nexus | No |  |
-| Quay | No | No
- 
- _If you are a registry or user and know that this page is out of date, [please
- let us know!](https://github.com/getporter/porter/issues/new)_
+| Quay | No | No |
+| Harbor | yes | yes |
```

Comment:
Thanks this reminds me that I copied this page over to https://cnab.io/registries, and that's where I updated it to say "Harbor 2". I forgot to update this original page as well.

Would you please edit this to specify the version of harbor **2** so that we are consistent across both pages?

Reply:
Done